wordpress.org/public_html/wp-content/plugins/wporg-gp-customizations/templates/js/editor.js

```diff
@@ -69,6 +71,48 @@
 		autosize( $textarea );
 	}
 
+	/**
+	 * Adds the suggestion to the translation in an array, and removes the previous suggestions, so
+	 * we only store the last one in the database, using the saveExternalSuggestions() function.
+	 *
+	 * @return {void}
+	 */
+	function addSuggestion() {
+		var $row = $(this);
+		var $originalId = $row.closest('tr').attr('id').substring(7);
+		var $CSSclass = $row.attr('class');
+		if ( $CSSclass.indexOf('openai') > -1 ) {
+			$openAITranslationsUsed[$originalId] = $row.find('.translation-suggestion__translation').text();
+			$deeplTranslationsUsed.splice($originalId, 1);
+		} else if ( $CSSclass.indexOf('deepl') > -1 ) {
+			$deeplTranslationsUsed[$originalId] = $row.find('.translation-suggestion__translation').text();
+			$openAITranslationsUsed.splice($originalId, 1);
+		}
+	}
+
+	/**
+	 * Saves the number of external suggestions used and used without modification.
+	 *
+	 * @return {void}
+	 **/
+	function saveExternalSuggestions() {
+		var $button = $(this);
+		var $row = $button.closest('tr.editor');
+		var $originalId = $row.attr('id').substring(7);
+		var $translation = $row.find('.lt-mirror__canvas').text().trim();
```

Comment:
That selector doesn't seem right. It causes for me an empty translation to be reported.

Reply:
Done. Thank you for catching this!